modules/ROOT/pages/manage-monitor-applications.adoc

```diff
@@ -4,11 +4,25 @@ Applications and API gateways deployed to Runtime Fabric include native support
 
 == Anypoint Monitoring
 
-xref:monitoring::index.adoc[Anypoint Monitoring] provides tools that give feedback from Mule flows and components 
+xref:monitoring::index.adoc[Anypoint Monitoring] includes tools that provide feedback from Mule flows and components 
 in your application network. For applications and API gateways deployed to Runtime Fabric, Anypoint Monitoring 
-provides basic metrics to all customers. Titanium customers have access to additional capabilities including 
+provides basic metrics to all customers. Titanium customers have access to additional capabilities, including 
 application log retrieval and search.
 
+=== Enable Anypoint Monitoring
```

Comment:
In most deliverables, I would expect task topics to not be "buried" at the third level of the information hierarchy.
I think we want "The User Journey" of tasks to be primary.